### PR TITLE
[DPM-149] casino back metrics and offset saving fix

### PR DESCRIPTION
**Pull request by justefg**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -73,19 +82,23 @@ type EventListener interface {
...
-		EventMessages: eventMessages, AppConfig: cfg}
+		EventMessages: eventMessages, AppConfig: cfg, Metrics: GetMetrics()}
 }
 
 func (app *App) processEvent(event *broker.Event) *string {
 	log.Debug().Msgf("Processing event %+v", event)
+	start := time.Now()
+	defer func() {
+		elapsed := time.Since(start)
+		app.Metrics.SigniDiceProcessingTimeMs.Observe(elapsed.Seconds() * 1000)
```
Comment (algys) [inline app.go:96]: why not `Milliseconds()`  ?

File: configs/config.dev.toml [hunk 2]
```diff
@@ -4,18 +4,18 @@ logLevel = "debug"
...
+platformPubKey = "EOS51QLenoFHatEB6WxLh5c7aRYfDKKFNW8c9NT7EMC9u7or2CMVo"
```
Comment (algys) [inline configs/config.dev.toml:18]: no need to commit keys....

**Review by algys — CHANGES_REQUESTED** [review]

File: app.go (hunk 1 above)
Comment (justefg) [reply app.go:96]: cause it returns an Int
but Seconds() returns a float

File: configs/config.dev.toml (hunk 2 above)
Comment (justefg) [reply configs/config.dev.toml:18]: these are not actual keys that are used in production, they're just there for validation to work

**Review by algys — APPROVED** [review]

**Pull request merged**